selftests/functional/test_basic.py

```diff
@@ -1,3 +1,4 @@
+# This Python file uses the following encoding: utf-8
```

Comment:
This is following the PEP263. There, we read: 
```
...
More precisely, the first or second line must match the regular
expression "^[ \t\v]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)".
...
```
And the line was extracted from the second example on the same PEP:
```
    2. Without interpreter line, using plain text:

          # This Python file uses the following encoding: utf-8
          import os, sys
          ...
```

The notation you pointed as 'more standard' is not described in PEP263 as so. But yes, it is a _de facto_ standard and I can use it to make you happy :)